internal/snmp/wrapper.go

```diff
@@ -167,7 +167,15 @@ func (gs *GosnmpWrapper) SetAgent(agent string) error {
 	switch u.Scheme {
 	case "tcp":
 		gs.Transport = "tcp"
-	case "", "udp":
+	case "tcp4":
+		gs.Transport = "tcp4"
+	case "tcp6":
+		gs.Transport = "tcp6"
+	case "udp4":
+		gs.Transport = "udp4"
+	case "udp6":
+		gs.Transport = "udp6"
+	case "udp":
 		gs.Transport = "udp"
```

Comment:
Maybe keep a more logical ordering?
```suggestion
	case "udp":
		gs.Transport = "udp"
	case "udp4":
		gs.Transport = "udp4"
	case "udp6":
		gs.Transport = "udp6"
```